.github/workflows/ci.yml

```diff
@@ -49,6 +49,51 @@ jobs:
 
       - run: |
           git ls-files | grep \\\.php$ | grep -v ^dictionaries/scripts/* | ./vendor/bin/parallel-lint --stdin
+
+  code-style:
+    name: Code Style Analysis
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
```

Comment:
Unlike other jobs, this is safe. We have 7.4 pinned in our `phpcs.xml`

Reply:
Ok, I'm switching back to 7.4.